sw/device/tests/aon_timer_wdog_bite_reset_test.c

```diff
@@ -54,15 +54,18 @@ static void wdog_bite_test(const dif_aon_timer_t *aon_timer,
   uint64_t bite_time_us = bark_time_us * 2;
   config_wdog(aon_timer, pwrmgr, bark_time_us, bite_time_us);
 
+  // The `intr_state` takes 3 aon clock cycles to rise, adding 5 cycles.
+  const uint32_t kWait = bark_time_us + 5 * 1000000 / kClockFreqAonHz;
```

Comment:
I guess the extra 2 cycles are because of CDC delays? Maybe it's worth explaining explicitly how we get from 3 to 5? Also, do you want to do something like `(5 * 1000000 + kClockFreqAonHz - 1) / kClockFreqAonHz` to make sure the division rounds up?

Reply:
Neat!!